commit a4cbbd907fa765a9115cc986accf414a53e39e7f
Author: Trent Mick <trentm@gmail.com>
Date:   2019-09-16T15:57:47+00:00 (3 weeks ago)
    
    TOOLS-2254 add redirects from mo-docs to modocs.joyent.us (restdown files should now always be .md)
    Reviewed by: Tim Foster <tim.foster@joyent.com>
    Approved by: Tim Foster <tim.foster@joyent.com>

diff --git a/prototypes/prototype.restdown b/prototypes/prototype.restdown
deleted file mode 100644
index 3bec307..0000000
--- a/prototypes/prototype.restdown
+++ /dev/null
@@ -1,15 +0,0 @@
----
-title: Something nice
-markdown2extras: wiki-tables, code-friendly
-apisections: Something, Useful
----
-<!--
-    This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at http://mozilla.org/MPL/2.0/.
--->
-
-<!--
-    Copyright 2019 Joyent, Inc.
--->
-
